reactjs/reactjs.org
# [Beta] Adds eslint integration on Sandpack

This introduces the integration between eslint and CodeMirror lint on Sandpack.

**How to test**
1. Enable feature flag `localStorage['SANDPACK_ESLINT'] = true`
2. Access the page example `/eslint` to see some example errors;

**Todo**
- [x] What are the eslint rules we want to implement? All of them or only react-hooks?
- [x] Proper place and design for list of errors;
- [x] Lazily load the eslint modules, but how?;
- [x] Should it work on all sandboxes or only specific ones? We could implement something like `showDevTools`;
- [x] Remove feature flag

<img width="993" alt="Screenshot 2022-05-17 at 21 25 47" src="https://user-images.githubusercontent.com/4838076/168903822-54e34067-efac-49d2-a993-ec2795fc305d.png">
